Dockerfile

```diff
@@ -1,56 +1,53 @@
 # syntax=docker/dockerfile:1.4
 
 ####################################################################################################
-## Prepare Workspace
+## Build Packages
 
-FROM node:18-alpine AS workspace
-WORKDIR /workspace
+FROM node:18-alpine AS builder
+WORKDIR /directus
+
+ENV NODE_OPTIONS=--max-old-space-size=8192
 
-COPY pnpm-lock.yaml .
 RUN corepack enable && corepack prepare pnpm@7.30.0 --activate
```

Comment:
Layer optimization; `corepack enable ....` and `ENV` should be set as a layer *before* the `pnpm fetch` which changes more frequently.